BinaryToOctalConvertor.java

```diff
@@ -0,0 +1,59 @@
+/**
+ *  To convert binary numbers to octal
+ *  
+ *  version 1.0
+ *  @author Udit Saxena
+ *
+ */
+
+public class BinaryToOctalConvertor {
+	/** 
+	 * * This function convert binary to octal
+	 * first it convert binary to decimal then decimal to octal
```

Comment:
Instead of converting binary to decimal then octal, We can directly convert binary to octal